.github/CODE_REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,83 @@
+# Code Review Guidelines  
+
+## PyGuide  
+
+- [Google guide](https://google.github.io/styleguide/pyguide.html)
+- [Antipatterns](https://docs.quantifiedcode.com/python-anti-patterns/index.html)
+- [Zen of python](https://zen-of-python.info/)
+- [Code review for humans](https://phauer.com/2018/code-review-guidelines/)
+- [Code review culture](https://www.giladpeleg.com/blog/better-code-review/)
+
+## General PR process  
+
+- The scope of PR should be simple, unique and well-defined. PR should not contain unrelated changes
+- Approve PR only if you are sure about the scope
+- Be respectful and reply asap
+- Do not merge PR without approval
+- Avoid spending too much time on trivial changes
+- Avoid premature optimisation
+- For community PR, check CLA by typing `recheckcla`
+- Ask for jina env if needed by `jina -vf`
+- Suggest changes and check back ASAP. Get PR merged soon
+- For new features, suggest adding appropriate documentation
+- Don&#39;t forget to praise when PR is ready with something like LGTM
```

Comment:
what's this weird character? some encoding problem?

Reply:
Nice catch; I converted the doc to markdown through an online converter which does this for `'`